javascript/javascript1/week2/event-application.js

```diff
@@ -0,0 +1,19 @@
+//Fvent application
+//Today is Sunday and the event is in 5 days. Therefore the event will be held on a friday.
+// Hint: use remainder operator, array indexes and investigate new Date in js.
+
+
+const weekDays = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"];
+
+function getEventWeekday (daysLeft) {
+  const today = new Date ();
+  const result  = today.getDay () + (daysLeft % 7);
```

Comment:
- You should add the today index to daysLeft then use `%`.
- This variable is not the result. It could be day index
```js
const dayIndex = (today.getDay() + daysLeft) % 7;
```

- In addition, don't add empty space before function parenthesis 
```js
today.getDay () // today.getDay()
```
If you use [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode)  it will help you with formatting your code